app/Http/Controllers/Api/V1/Project/RevisionController.php

```
@@ -139,7 +138,7 @@ public function changeToProposal(ApproveProposalRequest $request)
         DB::transaction(function () use ($validatedData) {
             ProjectService::changeToProposal($validatedData['id']);
             $revision = Project::findOrFail($validatedData['id']);
-            $revision->status = ProjectStage::PROPOSAL->value;
+            $revision->status = ProjectStatus::PENDING->value;
             $revision->save();
         });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'changeToProposal'. (Clean Code Rules)

(StaticAccess)